config/storm_email_to_username_mappings.yml

```diff
@@ -0,0 +1,2 @@
+agency-user@wcr.gov.uk: CarrierTest
```

Comment:
Isn't this for temporary local use only? In which case do not commit

Reply:
This is also for use in the development environment